README.md

```diff
@@ -449,6 +449,11 @@ only change from existing Kubernetes is that the *contents* of the
 
 If you want `SealedSecret` controller to take management of an existing `Secret` (i.e. overwrite it when unsealing a `SealedSecret` with the same name and namespace), then you have to annotate that `Secret` with the annotation `sealedsecrets.bitnami.com/managed: "true"` ahead applying the [Usage](#usage) steps.
 
+
+### Seal secret which can skip set owner references
+
+If you want `SealedSecret` and the `Secret` to be independent (which mean when you delete the `SealedSecret` the `Secret` won't disappear with). then you have to annotate that Secret with the annotation `skip-set-owner-references/skip-set-owner-references: "true"` ahead applying the Usage steps. This still add `sealedsecrets.bitnami.com/managed: "true"` to your `Secret` so that your secret will be updated when `SealedSecret` is updated. 
```

Comment:
```suggestion
If you want `SealedSecret` and the `Secret` to be independent, which mean when you delete the `SealedSecret` the `Secret` won't disappear with it, then you have to annotate that Secret with the annotation `sealedsecrets.bitnami.com/skip-set-owner-references: "true"` ahead of applying the Usage steps. You still may also add `sealedsecrets.bitnami.com/managed: "true"` to your `Secret` so that your secret will be updated when `SealedSecret` is updated. 
```

Reply:
So `sealedsecrets.bitnami.com/managed: "true"` should be added by the user instead of automatically when Unsealing? If so I should make an update for this.

Reply:
I would keep both tags separate and explicit, yes.